frontend/server/src/DAO/ProblemsetIdentities.php

```diff
@@ -158,7 +159,7 @@ public static function searchUsers(
                         i.name LIKE CONCAT("%", ?, "%")
                     );';
 
-        /** @var list<array{access_time: \OmegaUp\Timestamp|null, country_id: null|string, end_time: \OmegaUp\Timestamp|null, username: string}> */
+        /** @var list<array{access_time: \OmegaUp\Timestamp|null, country_id: null|string, end_time: \OmegaUp\Timestamp|null, username: string, name: string}> */
```

Comment:
huh! por qué esto te lo está marcando como `string`? debería ser `null|string` porque esa columna es nullable. se enoja si lo cambias por:

```suggestion
        /** @var list<array{access_time: \OmegaUp\Timestamp|null, country_id: null|string, end_time: \OmegaUp\Timestamp|null, username: string, name: null|string}> */
```

?

Reply:
Con qué comando consigues que te muestre estos errores?
https://travis-ci.com/github/omegaup/omegaup/jobs/328451609#L628

Reply:
`./stuff/mysql_types.sh`. tarda varios minutos :(